low-level/framework/utils/consulstore.py

```diff
@@ -60,8 +60,8 @@ def put(self, value, key, pickled=True):
             try:
                 key = self._get_key(key)
                 if pickled:
-                    value = pickle.dumps(value)
```

Comment:
Observed that storing pickled value in the same variable causing data corruption when consul is down and if suddenly it comes back that time it will pickle value again which is already pickled.

Reply:
Good catch @VijayThakkar132